.travis.yml

```diff
@@ -63,7 +63,7 @@ env:
     - "TESTLANG=Lua"
     - "TESTLANG=Mumps"
     - "TESTLANG=Nim"
-    - 'TESTDIR="Nim/httpbeast Nim/jester Nim/basolato"'
+    - 'TESTDIR="Nim/httpbeast Nim/jester Nim/basolato Nim/prologue"'
```

Comment:
Also, please remove this line. The line above it tests all Nim frameworks

Reply:
It was there previously as well. But I will remove the 2nd line.

Reply:
Done